src/VisualStudio/Core/Def/Implementation/Watson/WatsonReporter.cs

```diff
@@ -18,44 +15,20 @@ internal class WatsonReporter
         /// <param name="exception">Exception that triggered this non-fatal error</param>
         public static void Report(Exception exception)
         {
-            // Log the exception regardless of whether the Watson will be throttled.
-            Logger.Log(FunctionId.NonFatalWatson, KeyValueLogMessage.Create(dict =>
-            {
-                dict["ExceptionType"] = exception.GetType().FullName;
-                dict["ExceptionStack"] = GetStackTraceString(exception);
-            }));
-
-            using (var report = WatsonErrorReport.CreateNonFatalReport(new ExceptionInfo(exception, "Roslyn")))
-            {
-                // Ignore the return value on purpose, we don't care if it actually gets submitted
-                report.ReportIfNecessary();
-            }
+            Report("Roslyn NonFatal Watson", exception);
```

Comment:
it just saved as one of PX information along with the watson such as this

P1: Microsoft.Windows.Cortana_1.7.0.14393_neutral_neutral_cw5n1h2txyewy
P2: praid:CortanaUI
P3: 10.0.14393.321
P4: 57f4c5cf
P5: SearchUI.exe
P6: 10.0.14393.321
P7: 57f4c5cf
P8: c0000005
P9: 00000000001bf061
P10: 

but I dont remember exactly what number of PX value it goes to.